pages/issuing/cardholder_terms.en.md

```diff
@@ -0,0 +1,176 @@
+
+For security reasons, we may limit the amount or number of transactions you can make on your Card. Your Card cannot be redeemed for cash. You may not use your Card for online gambling or any illegal transaction.
+
+Each time you use your Card, you authorize us to reduce the value available in your Card Account by the amount of the transaction and any applicable fees. You are not allowed to exceed the available amount in your Card Account through an individual transaction or a series of transactions.
+
+Nevertheless, if a transaction exceeds the balance of the funds available on your Card, you shall remain fully liable to us for the amount of the transaction and any applicable fees.
+
+You do not have the right to stop payment on any purchase or payment transaction originated by use of your Card. If you authorize a transaction and then fail to make a purchase of that item as planned, the approval may result in a hold for that amount of funds for up to thirty (30) days. 
+
+### Returns and Refunds
+
+If you are entitled to a refund for any reason for goods or services obtained with your Card, you agree to accept credits to your Card for such refunds and agree to the refund policy of that merchant. The Issuer is not responsible for the delivery, quality, safety, legality or any other aspects of goods or services that you purchase from others with a Card. All such disputes must be addressed and handled directly with the merchant from whom those goods or services were provided.
+
+### Card Replacement
+
+If you need to replace your Card for any reason, please return to a branch of the financial institution where the Card was purchased to obtain an immediate replacement, or contact us at +855(0)23 532 0888 to request a replacement Card be mailed to you. You will be required to provide personal information which may include your Card number, full name, transaction history, etc. There is a fee for replacing your Card. Refer to the fee schedule provided within this document.
```

Comment:
This section should mention that they should order a card replacement via the app.